pubsub/cloud-client/README.md

```diff
@@ -15,12 +15,18 @@ Install [Maven](http://maven.apache.org/).
 Build your project with:
 
 	mvn clean package -DskipTests
-	
-You can then run a given `ClassName` via:
 
-	mvn exec:java -Dexec.mainClass=com.example.pubsub.ClassName \
-	    -DpropertyName=propertyValue \
-		-Dexec.args="any arguments to the app"
+## Testing
+
+To run the tests for this sample, first set the `GOOGLE_CLOUD_PROJECT`
+environment variable. The project should have a dataset named `test_dataset`
+with a table named `test_table`.
```

Comment:
Is this the right README change? Isn't pubsub organized by topic, rather than dataset/table?